src/gui/owncloudsetupwizard.cpp

```diff
@@ -119,6 +119,7 @@ void OwncloudSetupWizard::startWizard()
         localFolder = QDir::homePath() + QLatin1Char('/') + localFolder;
     }
 
+    localFolder = "Please select folder";
```

Comment:
I plan to add a few more lines of code so that the general user gets the default folder as usual. Only in special cases do I want to force the user to manually select a folder without a default (or an institution-specific default).